### PR TITLE
Correctly format goreleaser-cdk

### PR DESCRIPTION
**Pull request by vcastellm**
Fix command

### CODE REVIEW

**Review by goran-ethernal — APPROVED** [review]

File: .goreleaser-cdk.yaml [hunk 1]
```diff
@@ -10,7 +10,8 @@ before:
   hooks:
     - go mod download
     - go install github.com/gobuffalo/packr/v2/packr2@v2.8.3
-    - ( cd db && packr2 )
+    - cd db
+    - packr2
```
Comment (christophercampbell) [inline .goreleaser-cdk.yaml:14]: I was under the impression that this command would not be run in the db directory, but maybe it just works?

Comment (vcastellm) [reply .goreleaser-cdk.yaml:14]: just works, I checked 🤷

Comment (christophercampbell) [reply .goreleaser-cdk.yaml:14]: awesome, I did too, and I think we can drop the `cd db` part, and then can we just .gitignore the packr2 generated code?

Comment (vcastellm) [reply .goreleaser-cdk.yaml:14]: nono, without the `cd db` it doesn't work, I'll merge this as is, gen image and open a PR in node replacing packr2

**Review by christophercampbell — APPROVED** [review]

**Pull request merged**